### PR TITLE
Issue#135 interactive programs

### LINKED ISSUE #135: Support interactive programs in Kids mode

### PR DESCRIPTION
**Pull request by luchotc**
Resolves #135

### CODE REVIEW

File: lib/render/editor/editor.html [hunk 1]
```diff
@@ -459,13 +533,15 @@
...
 
-            this._onRemoteExecutionStop(results)
+            this._onRemoteExecutionStop(results);
           })
           .catch(() => {
             mumuki.kids.showResult({status: 'aborted'})
           });
         this.serverPromise = promise;
 
+        editor.toggleInteractiveMode();
```
Comment (rodri042) [inline lib/render/editor/editor.html]: Should we call this only if `editor.interactiveMode`?

**Comment by rodri042** [comment]
I don't like the fact that `editor.html` is getting huge as we add new features. I'd like to split it, but right now it's kind of complicated since we need to:
- update here https://github.com/mumuki/mumuki-gobstones-runner/blob/master/lib/assets_server.rb#L45
- update https://github.com/mumuki/mumuki-gobstones-runner/blob/master/lib/metadata_hook.rb#L23
- reimport the language in laboratory
- and I don't know if there are places like bibliotheca or classroom where we're importing the editor.html directly.

So maybe another solution could be compiling several files in one (like `vulcanize` does). I'll create another issue for that.

File: lib/render/editor/editor.html (hunk 1 above)
Comment (luchotc) [reply lib/render/editor/editor.html]: I could add an if here. It doesn't affect the current layout, as it is just toggling .play-mode that doesn't change anything without .mu-kids-interactive

**Comment by flbulgarelli** [comment]
Looks good, just please import hammer directly from the other assets, so that no extra import on laboratory is required.

**Pull request merged**